Keyboards/KeyboardsBase/KeyboardViewController.swift

```diff
@@ -2657,6 +2684,15 @@ class KeyboardViewController: UIInputViewController {
     }
   }
 
+  private func switchToFullCaps() {
```

Comment:
Another thing that we've been dealing with is that not enough of the code base is split into functions. I do some rounds from time to time and break some things out, but if you came across some parts where we can extract some code then by all means let me know or open an issue!